tests/openvino/native/quantization/test_quantization_pipeline.py

```diff
@@ -103,6 +103,9 @@ def check_parameters(quantized_model, parameters, path):
             if isinstance(value, IgnoredScope):
                 check_parameters(quantized_model, value.__dict__, rt_path)
                 continue
+            if "ignored_scope" in rt_path:
+                assert key != "validate"  # key validate shouldn't be dumped
+                assert value  # empty values shouldn't be dumped
```

Comment:
It doesn't check the content of actual dumped runtime info, instead it directly accesses the input parameters. 
Please add a unit test checking the content of <rt_info> for a dummy synthetic model, for example via get_rt_info.